packages/custom-functions-metadata/Specification.md

```diff
@@ -32,6 +46,13 @@ If the last function parameter is of type `CustomFunctions.CancelableInvocation`
 
  A function cannot have both `@cancelable` and `@streaming` tags.
 
+---
+### @capturesCallingObject
+
+Indicates that the custom function captures the calling object i.e. the [Excel.CellValue](https://learn.microsoft.com/es-es/javascript/api/excel/excel.cellvalue)
+object that invoked the custom function, in the first parameter.
+
+This tag is required to generate metadata for the custom function.
```

Comment:
Please improve the wording for "This tag is required to generate metadata for the custom function."